create_jenkins_job.py

```diff
@@ -186,13 +192,24 @@ def create_job(os_name, job_name, template_file, additional_dict):
 
         # configure nightly job for compiling with clang on linux
         if os_name == 'linux':
-            create_job(os_name, 'nightly_' + os_name + '_clang', 'ci_job.xml.em', {
+            create_job('linux', 'nightly_linux_clang', 'ci_job.xml.em', {
                 'cmake_build_type': 'Debug',
                 'compile_with_clang_default': 'true',
                 'time_trigger_spec': PERIODIC_JOB_SPEC,
                 'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
             })
 
+        # configure nightly job for testing rmw/rcl based packages with address sanitizer on linux
+        if os_name == 'linux':
+            create_job('linux', 'nightly_linux_address_sanitizer', 'ci_job.xml.em', {
+                'cmake_build_type': 'Debug',
+                'enable_sanitizer_type_default': 'address',
+                'time_trigger_spec': PERIODIC_JOB_SPEC,
+                'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
+                'test_args_default': '--event-handlers console_direct+ --executor sequential --packages-up-to ' + ' '.join(RMW_RCL_TEST_PKGS),
```

Comment:
Instead of copy-n-pasting the list of arguments please use the existing variable and only list the additional arguments here.

Same next line.

Reply:
I am avoiding the "retest until pass" arg from the default set of args as this might unnecessarily slow down asan/tsan test runs